### ISSUE TITLE
Unable to use SSL with Apache24

### ISSUE THREAD

**Issue by lattera** [HardenedBSD/hardenedbsd-ports#62]
```
httpd: Syntax error on line 132 of /usr/local/etc/apache24/httpd.conf: Cannot load libexec/apache24/mod_ssl.so into server: /usr/local/libexec/apache24/mod_ssl.so: Undefined symbol "X509_up_ref"
```

No OpenSSL package installed, using LibreSSL in base on HardenedBSD 12-CURRENT/amd64.

**Comment by lattera** [comment]
@Sp1l please consider this issue as urgent.

**Comment by Sp1l** [comment]
Closed by 00898a0ddb1d4683b246ba12525dc8f453dee6ec
